design-system/molecules/avatar-label-group/adapters/default/default.adapter.tsx

```diff
@@ -30,6 +30,21 @@ export function AvatarLabelGroupDefaultAdapter<C extends ElementType = "div">({
 
   const slots = AvatarLabelGroupDefaultVariants({ truncate });
 
+  let titleSize: TypoSize = "sm";
+  let descriptionSize: TypoSize = "sm";
+
+  switch (size) {
+    case "md":
+    case "sm":
+    case "xs":
+      descriptionSize = "xs";
+      break;
+    case "xxs":
+      titleSize = "xs";
+      descriptionSize = "xs";
+      break;
+  }
```

Comment:
```suggestion
  const {
		titleSize,
		descriptionSize,
	} = useMemo(() => {

		switch (size) {
			case "md":
			case "sm":
			case "xs":
				return {
					titleSize: "sm",
				descriptionSize: "xs",
				}
			case "xxs":
				return {
					titleSize: "xs",
					descriptionSize: "xs",
				}
		}
	}, [size]);
```

Do not take it exactly it's just an exemple